02-Git: Add homework application

Serggio-26 opened the pull request:
The simple CLI rock-paper-scissors game

Signed-off-by: Sergey Dyachenko <Sergey.Dyachenko@gmail.com>

Serggio-26 committed:
02-Git: Add homework application

The simple CLI rock-paper-scissors game

Signed-off-by: Sergey Dyachenko <Sergey.Dyachenko@gmail.com>

an1kh reviewed (changes requested):
Please add extra commits. Not just one.
E.g. you may add random CPU-side choice.

an1kh commented on 02-Git/Makefile:
```diff
@@ -0,0 +1,15 @@
+	-rm ./$(BUILD_DIR)/$(TARGET)
+
+run:
+	./$(BUILD_DIR)/$(TARGET)
```
No empty line at the end of the file

Serggio-26 committed:
02-Git: Change game algorithm

Game algorithm is changed to let player ability to win. Computer
chooses its bet rendomly.

Signed-off-by: Serhii Diachenko <Sergey.Dyachenko@gmail.com>

an1kh commented on 02-Git/Makefile:
```diff
@@ -12,4 +12,5 @@ clean:
 	-rm ./$(BUILD_DIR)/$(TARGET)
 
 run:
-	./$(BUILD_DIR)/$(TARGET)
```
Please do not do fixes to not merged patches!
Squash this commit with the first one.

an1kh commented on 02-Git/Makefile:
```diff
@@ -12,4 +12,5 @@ clean:
-	./$(BUILD_DIR)/$(TARGET)
\ No newline at end of file
+	./$(BUILD_DIR)/$(TARGET)
+	
```
Still saying no empty line

an1kh reviewed (approved):
Well done! Please merge
